app/controllers/my_facilities_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class MyFacilitiesController < AdminController
+  include DistrictFiltering
+  include Pagination
+  include SizeFiltering
+
+  before_action :authorize_my_facilities, only: %i[index ranked_facilities blood_pressure_control]
+
+  def index
+    @users_requesting_approval = policy_scope([:manage, :user, User])
+                                 .requested_sync_approval
+                                 .order(updated_at: :desc)
+
+    @users_requesting_approval = paginate(@users_requesting_approval)
+
+    @facilities = policy_scope([:manage, :facility, Facility])
+    @inactive_facilities = MyFacilitiesQuery.inactive_facilities(@facilities)
+
+    @facility_counts_by_size = { total: @facilities.group(:facility_size).count,
+                                 inactive: @inactive_facilities.group(:facility_size).count }
```

Comment:
Can we make _all_ of the queries in this controller through the `MyFacilitiesQuery` class? It seems like a great place to put them, since the purpose of that class is to encapsulate queries needed to populate the My Facilities dashboard.

Reply:
We're trying to follow the QueryObject pattern for the new query classes, so the idea is to only return relations from the QueryObject itself. This means we'll have to do grouping, counting etc in the controller. We might consider moving these things to a ViewObject if it gets too hard to keep track of.